dotnet/src/Experimental/Assistants/AssistantResponse.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Experimental.Assistants;
+
+/// <summary>
+/// Response from assistant when called as a <see cref="ISKFunction"/>.
+/// </summary>
+public class AssistantResponse
```

Comment:
Should this be public? Not a big deal now as this is experimental but one to watch later.

Reply:
I believe it might need to be consumed by callers for certain scenarios, yes.  That's why I moved it out of the models folder.  Agree this needs to be watched.